packages/credential-registry/schemas/ClaimedCredential.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ClaimedCredential",
+  "description": "A [[ClaimedCredential]] represents a specific, factually-oriented claim that could be the Id and CredentialSubject of a [[VerifiableCredential]]",
+  "type": "object",
+  "properties": {
+    "did": {
+      "type": "string"
+    },
+    "type": {
+      "type": "string"
+    },
+    "typeSchema": {
+      "type": "string"
+    },
+    "value": {
+      "type": "object",
+      "properties": {
+        "keccak256": { "type": "string" },
+        "value": { "type": "string" },
+        "description": { "type": "string" },
+        "country": { "type": "string" },
+        "city": { "type": "string" },
+        "postalCode": { "type": "string" },
+        "date": { "type": "string" },
+        "time": { "type": "string" },
+        "startDate": { "type": "string" },
+        "endDate": { "type": "string" },
+        "issuanceDate": { "type": "string" },
+        "expirationDate": { "type": "string" },
+        "issuingEntity": { "type": "string" },
+        "imageIPFS": { "type": "string" }
+      }
+    },
+    "encrypted": {
+      "$ref": "#/definitions/JWE"
+    },
+    "verifiableCredentials": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/CeramicDocId"
```

Comment:
```suggestion
        "$ref": "#/definitions/CeramicStreamId"
```

Reply:
There are many places throughout this PR that refer to a Ceramic DocID.  They should all be updated to StreamId as that is the current terminology used by Ceramic